### PR TITLE
Change space desc, change activities footer text, change no activitie…

### PR DESCRIPTION
**Pull request by AlexAndBear**
…s text

## Related Issue

- Fixes https://github.com/owncloud/web/issues/11106

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-files/src/components/SideBar/ActivitiesPanel.vue [hunk 1]
```diff
@@ -55,9 +55,9 @@ export default defineComponent({
     const activities = ref<Activity[]>([])
     const activitiesLimit = 200
 
-    const activitiesLimitText = computed(() => {
-      return $gettext('(Showing latest %{activitiesLimit} activities)', {
-        activitiesLimit: activitiesLimit.toString()
+    const activitiesFooterText = computed(() => {
+      return $gettext('%{activitiesCount} activities in total', {
```
[the remark this answers is not in the document]
Comment (AlexAndBear) [reply packages/web-app-files/src/components/SideBar/ActivitiesPanel.vue:59]: fuck yes, toby provided me this information :/ any fast suggestion, or should I remove it from the pr?

[the remark this answers is not in the document]
Comment (kulmann) [reply packages/web-app-files/src/components/SideBar/ActivitiesPanel.vue:59]: `Showing %{activitiesCount} activities`

[the remark this answers is not in the document]
Comment (kulmann) [reply packages/web-app-files/src/components/SideBar/ActivitiesPanel.vue:59]: oh, and this needs to be `$ngettext` 🙈

[the remark this answers is not in the document]
Comment (AlexAndBear) [reply packages/web-app-files/src/components/SideBar/ActivitiesPanel.vue:59]: THX!

File: packages/web-app-files/src/components/SideBar/ActivitiesPanel.vue [hunk 2]
```diff
@@ -55,9 +55,9 @@ export default defineComponent({
     const activities = ref<Activity[]>([])
     const activitiesLimit = 200
 
-    const activitiesLimitText = computed(() => {
-      return $gettext('(Showing latest %{activitiesLimit} activities)', {
-        activitiesLimit: activitiesLimit.toString()
+    const activitiesFooterText = computed(() => {
+      return $gettext('Showing %{activitiesCount} activities', {
```
Comment (kulmann) [inline packages/web-app-files/src/components/SideBar/ActivitiesPanel.vue:59]: Please use `$ngettext` and two strings, one for singular and one for plural.

Comment (AlexAndBear) [reply packages/web-app-files/src/components/SideBar/ActivitiesPanel.vue:59]: on it my fren!!!!!

**Review by kulmann — APPROVED** [review]

**Pull request merged**